### DISCUSSION TITLE
Build 5.23.1601: Auswahllisten bei Darstellungsfiltern mit "NULL"-Values befüllt

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#70] [General]
Die Auswahllisten sind seit dem UPDATE mit NULL-Values befüllt.  Die Anzahl der NULL-Values entspricht der Anzahl der Werte die in der Auswahlliste angezeigt werden sollen - daher kein Datenverbindungsproblem. Es handelt sich um eine ORACLE-Connection.

Die gleichen Datenverbindungen funktionieren im Bereich Auswahllisten/Editing und Datalinq.  Wir sind wieder auf Build 5.22.5002 gegangen - hier tritt dieses Verhalten nicht auf.

**Comment by gis-eni** [comment]
Hallo Christian,
Das Problem ist bekannt. Andreas hat das schon gemeldet, allerdings per Mail, drum hat es hier wahrscheinlich keiner mitbekommen ;)
Der Fehler wurde bereits behoben und der neue Build liegt bereits am Server.

**Comment by cmairamhof** [comment]
Danke
